crates/recursor/src/lib.rs

```diff
@@ -54,6 +56,15 @@ pub enum DnssecPolicy {
     /// DNSSEC validation is disabled; DNSSEC records will be requested and processed
     #[cfg(feature = "dnssec")]
     ValidationDisabled,
+
+    /// DNSSEC validation is enabled and will use the chosen `trust_anchor` set of keys
+    #[cfg(feature = "dnssec")]
+    ValidateWithStaticKey {
+        /// set to `None` to use built-in trust anchor
+        trust_anchor: Option<TrustAnchor>,
```

Comment:
I like that idea.

It does require changes elsewhere because this `trust_anchor` is used to initialize a `DnssecDnsHandle` and its constructor expects a `TrustAnchor` by value ... which then immediately wraps in an `Arc` for storage in one of its fields.

I submitted the `Arc` changes in a separate commit